src/main/java/org/isf/admission/dto/AdmissionDTO.java

```diff
@@ -90,14 +90,16 @@ public class AdmissionDTO {
 
 	@ApiModelProperty(notes = "operation date", position = 15)
 	private LocalDateTime opDate;
+	//@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```

Comment:
@ArnaudFonzam do you agree we don't need anymore this annotations?

Reply:
yes we can remove it